docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+version: "3"
+
+volumes:
+  postgres_data:
+    driver: local
+
+services:
+  postgres:
+    image: postgres
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_DB: keycloak
+      POSTGRES_USER: LFY_POSTGRES_KEYCLOAK_USER
+      POSTGRES_PASSWORD: "SajCasdk]=V"
+  keycloak:
+    image: jboss/keycloak:12.0.4
+    volumes:
+      - /Users/kamil/IdeaProjects/keycloak-theme/theme/lottify-theme:/opt/jboss/keycloak/themes/lottify-theme
```

Comment:
I think file shouldn't be checked in, or at least not in this form with references to "LFY" and including the actual passwords we use for local development. What do you think?

Reply:
ahh yes, I will remove this file